bare-metal/elastic-metal/reference-content/shared-responsibility-model.mdx

```diff
@@ -0,0 +1,118 @@
+---
+meta:
+  title: Scaleway Bare Metal Shared Responsibility Model
+  description: Learn about the shared responsibility model for Scaleway Bare Metal services, outlining the roles of Scaleway and users in managing server security, backups, and compliance.
+content:
+  h1: Scaleway Bare Metal Shared Responsibility Model
+  paragraph: Learn about the shared responsibility model for Scaleway Bare Metal services, outlining the roles of Scaleway and users in managing server security, backups, and compliance.
+tags: bare metal shared responsibility
+dates:
+  validation: 2024-07-18
+  posted: 2024-07-18
+categories:
+  - bare-metal
+---
+
+On this page, we outline the roles and responsibilities for maintaining and securing your Bare Metal servers.
+Our shared responsibilty model clarifies the division of duties between Scaleway and our users, ensuring clarity in managing server availability, backups, configurations, and security measures. 
+By understanding this shared responsibility, you can optimize the performance, reliability, and security of your Scaleway Bare Metal services.
+
+## Product resiliency
+
+### Availability
+
+Bare Metal Products target the following power supplies and connectivity SLOs:
+
+* For Dedibox servers:
+    * Start: 99.9% availability
+    * Pro: 99.95% availability
+    * Core and Store: 99.99% availability
+* For Elastic Metal servers:
+    * Aluminium: 99.9% availability
+    * Beryllium and Lithium: 99.95% availability
+    * Iridium: 99.99% availability
+* For Apple silicon servers:
+    * Mac mini: 99.95% availability
+
+The detailed SLAs measurements and guarantees can be found at [Service Level Agreement for Bare Metal Services](https://www.scaleway.com/en/bare-metal/sla/).
+
+Once the resource is in your hands, we have no access to the machine and therefore no way to monitor its operation.
+In case of any operational issues, we invite you to consult our [documentation](/bare-metal/) or [create a ticket](https://console.scaleway.com/support/tickets/create) with our support team. We will then do our utmost to ensure that you regain access to your resource as quickly as possible.
```

Comment:
```suggestion
In case of any operational issues, we invite you to consult our [documentation](/bare-metal/) or [create a ticket](https://console.scaleway.com/support/tickets/create) with our support team. We will then do our utmost to ensure that you regain access to your resources as quickly as possible.
```